bin/travis-run28.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+echo "TESTING ckanext-datagovtheme 2.8"
+
+nosetests --ckan --with-pylons=test.ini ckanext/datagovtheme/tests/test_datagovtheme_28.py
```

Comment:
This file seems strange to me. Should we always be running _all_ the tests?

Reply:
I split the tests because one is in an environment using 2.3 ckan and the other is in an environment using 2.8 ckan so only the tests for the applicable environment should be run

Reply:
Is it possible to stub the is_bootstrap2 method, render the template, and test the output? Then you would have several tests, one that stubs the method as version 2.3, another as 2.8, even one as 2.6 or 2.9. And all should pass no matter what the dependencies are installed,  including  CKAN.

I know it sounds a bit weird but we should be running the same tests against both versions... we want to test that our code does the right thing no matter what dependencies are installed.

Reply:
They are essentially the same test - but even the test setup is different due to the ckan version - I can combine the tests but will have to use a lot of if ckan_version statements. Is that ok?